packages/base-dao/src/baseDAO.test.ts

```diff
@@ -0,0 +1,278 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
when this packages was switched to ts, these tests stopped running, because the test file wasnt in the correct convention... -test instead of .test -- when i fixed it now, i had to also handle some issues which are not really a part of this PR logically... like the daoAuth imports etc.

This file is full of tsignores etc... but i rather not invest time into it at the moment